### DISCUSSION TITLE
Inscription en cohorte

### DISCUSSION THREAD

**Discussion by LHuberson** [unistra/esup-immersup-demandes#7] [3- Création de compte pour venir en immersion]
Nous organisons des interventions dans  les établissements pour des groupes classes. Actuellement se sont les enseignants des lycées concernés qui inscrivent leurs classes par mail auprès des collègues. C'est très chronophage. Norial pourrait-elle permettre l'inscription à des créneaux horaires par un enseignant pour sa classe. Est -il possible de limiter cet accès à certains établissements?

**Comment by SandrineRobert** [comment]
Cette possibilité d'inscrire une classe serait intéressante pour nous également lorsque nous faisons venir sur une journée un lycée (prioritaire ACCES) à nos journées d'immersion.
@LHuberson je vous confirme qu'il est possible de bloquer un nombre de places pour un lycée en particulier. C'est ce que nous avions fait pour organiser l'immersion d'une classe qui venait sur une journée précise. Nous avions alors suggéré au lycée concerné de prévoir des temps dédiés en classe, avec le PP, pour que les élèves s'inscrivent aux-mêmes sur ces créneaux dédiés. Mais cela reste chronophage (pour le lycée et nous) pour vérifier que tous les élèves se sont bien inscrits au bon cours et à la bonne date !

**Comment by efinkbeiner5** [comment]
Mérite un atelier d'analyse

**Comment by efinkbeiner5** [comment]
Fait dans la version 4 en 2024
